src/core/config/descriptor_providers/xlio_config_schema.json

```diff
@@ -0,0 +1,1454 @@
+                                        "period_usec": {
+                                            "type": "integer",
+                                            "default": 50,
+                                            "title": "Moderation period (µsec)",
+                                            "description": "Period in micro-seconds for holding the packet before generating interrupt. Maps to XLIO_CQ_MODERATION_PERIOD_USEC environment variable."
+                                        },
+                                        "adaptive_count": {
+                                            "type": "integer",
+                                            "default": 500,
+                                            "title": "Adaptive moderation count threshold",
+                                            "description": "Maximum count value to use in the adaptive interrupt moderation algorithm. Maps to XLIO_CQ_AIM_MAX_COUNT environment variable."
+                                        },
+                                        "adaptive_period_usec": {
+                                            "type": "integer",
+                                            "default": 1000,
+                                            "title": "Adaptive moderation period (µsec)",
+                                            "description": "Maximum period value to use in the adaptive interrupt moderation algorithm. Maps to XLIO_CQ_AIM_MAX_PERIOD_USEC environment variable."
+                                        },
+                                        "adaptive_change_frequency_msec": {
+                                            "type": "integer",
+                                            "default": 1000,
+                                            "title": "Adaptive change frequency (msec)",
+                                            "description": "Frequency of interrupt moderation adaptation. Maps to XLIO_CQ_AIM_INTERVAL_MSEC environment variable. Interval in milliseconds between adaptation attempts. Use value of 0 to disable adaptive interrupt moderation."
+                                        },
+                                        "interrupt_per_sec": {
+                                            "type": "integer",
+                                            "default": 10000,
+                                            "title": "Target interrupts per second",
+                                            "description": "Desired interrupts rate per second for each ring (CQ). Maps to XLIO_CQ_AIM_INTERRUPTS_RATE_PER_SEC environment variable. The count and period parameters for CQ moderation will change automatically to achieve the desired interrupt rate for the current traffic rate."
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        "rx_spare_buffers": {
```

Comment:
maybe this one should be under ring...

Reply:
but it's CQ config, no?

Reply:
let's talk also about `network.multicast.mc_disable_flowtag`